release-notes/opensearch-release-notes-1.3.0.md

```diff
@@ -0,0 +1,632 @@
+# OpenSearch and Dashboards 1.3.0 Release Notes
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 1.3.0 includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.0.md).
+
+## FEATURES
+
+### Opensearch Anomaly Detection
```

Comment:
Should be `OpenSearch`.

Reply:
Lets rename everywhere from opensearch -> OpenSearch